public/scripts/adobe.js

```diff
@@ -0,0 +1,4 @@
+window.adobeDataLayer = adobeDataLayer
+_satellite.setVar('jsAbort', 'false')
+_satellite.setVar('isSpa', 'true')
```

Comment:
Did Jorge say your site wasn't whitelisted? If it was the the setVar instructions can be removed

Reply:
yea, I will confirm it. Forgot about it. Thanks

Reply:
just confirmed that it is white listed. So I removed two setVar lines.